src/commands/math/basicSymbols.js

```diff
@@ -140,7 +140,7 @@ var BuiltInOpNames = {}; // http://latex.wikia.com/wiki/List_of_LaTeX_symbols#Na
 var TwoWordOpNames = { limsup: 1, liminf: 1, projlim: 1, injlim: 1 };
 var AutoOpNames = Options.p.autoOperatorNames = { _maxLength: 9 };
 (function() {
-  var mostOps = ('arg deg det dim exp gcd hom inf ker lg lim ln log max min sup'
+  var mostOps = ('arg deg det dim exp gcd hom inf ker lg ln log max min sup'
```

Comment:
Oh, this is an unfortunate tradeoff I hadn't considered. So this now requires `lim` to be an `autoCommand` or you have to type a backslash? If it's an `autoCommand`, it has the unfortunate result that typing `l`, `i`, `m`, and `Backspace` deletes everything rather than leaving you with `li`, right? I guess `sum` as an `autoCommand` has the same problem, but at least ∑ doesn't look like 3 separate characters.

I think I was originally imagining that `lim` would still auto-unitalicize normally, and to get the special approaches blocks you'd type `_` after it, how would you feel about that? (When the special approaches blocks are active, the cursor can't go between the letters; but once you delete it, it'd go back to being 3 letters that happen to be unitalicized, just like any other auto-unitalicized command.)